lib/countly.js

```diff
@@ -1832,8 +1842,25 @@
         * @param {number} feedback.rating - user's rating from 1 to 5
         * @param {string=} feedback.email - user's email
         * @param {string=} feedback.comment - user's comment
+        * 
+        * @deprecated use 'recordRatingWidgetWithID' in place of this call
         **/
         this.report_feedback = function(feedback) {
+            log("report_feedback, Deprecated function call! Use 'recordRatingWidgetWithID' in place of this call. Call will be redirected now!");
+            this.recordRatingWidgetWithID(feedback);
+        };
+        /**
+        * Provide information about user
+        * @param {Object} feedback - object with feedback properties
+        * @param {string} feedback.widget_id - id of the widget in the dashboard
+        * @param {boolean=} feedback.contactMe - did user give consent to contact him
+        * @param {string=} feedback.platform - user's platform (will be filled if not provided)
+        * @param {string=} feedback.app_version - app's app version (will be filled if not provided)
+        * @param {number} feedback.rating - user's rating from 1 to 5
+        * @param {string=} feedback.email - user's email
+        * @param {string=} feedback.comment - user's comment
+        **/
+        this.recordRatingWidgetWithID = function(feedback) {
```

Comment:
In the function bellow, replace mentions and usages of "Feedback" to "Rating widget"